### PR TITLE
Add test for animated_align.0.dart API example.

### PR DESCRIPTION
**Pull request by ksokolovskyi**
This PR contributes to https://github.com/flutter/flutter/issues/130459

### Description
- Adds `examples/api/test/widgets/implicit_animations/animated_align.0_test.dart` test

## Pre-launch Checklist

- [X] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [X] I read the [Tree Hygiene] wiki page, which explains my responsibilities.
- [X] I read and followed the [Flutter Style Guide], including [Features we expect every widget to implement].
- [X] I signed the [CLA].
- [X] I listed at least one issue that this PR fixes in the description above.
- [ ] I updated/added relevant documentation (doc comments with `///`).
- [X] I added new tests to check the change I am making, or this PR is [test-exempt].
- [X] I followed the [breaking change policy] and added [Data Driven Fixes] where supported.
- [X] All existing and new tests are passing.

### CODE REVIEW

**Review by Renzo-Olivares — APPROVED** [review]
LGTM, thanks for the contribution!

File: examples/api/test/widgets/implicit_animations/animated_align.0_test.dart [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+      find.descendant(
+        of: find.byType(AnimatedAlign),
+        matching: find.byType(Align),
+      ),
+    );
+    expect(align.alignment, Alignment.bottomLeft);
+
+    // Advance animation to the end by the 1-second duration specified in
+    // the example app.
+    await tester.pump(const Duration(seconds: 1));
```
Comment (justinmc) [inline examples/api/test/widgets/implicit_animations/animated_align.0_test.dart:37]: Same nit here as in other PRs about referencing the duration from the example app and pumping half at a time to confirm animation happened.

**Review by justinmc — APPROVED** [review]
LGTM with nits 👍

**Comment by ksokolovskyi** [comment]
@justinmc @Renzo-Olivares thanks for the review!
I applied suggestions in the latest commit.

**Review by Renzo-Olivares — APPROVED** [review]

**Pull request merged**